User-Help/Voting.md

```diff
@@ -0,0 +1,40 @@
+| URL slug | Category |
+|:--------:|:--------:|
+| voting | Guidance |
+
+# Why should I vote?
+
+Voting is the primary indicator of valuable content.  Vote up posts that add to the community, and vote down ones that detract from it.  (Many posts do neither; you don't have to vote on everything.)
+
+Voting influences a post's score.  Post score contributes to earning abilities.  When you vote, you are, in a sense, saying "I would like the community to get more (or less) of this kind of post".
+
+
+# When should I vote?
+
+![voting buttons](/assets/voting.png]
+
+You can vote up or down on posts by clicking the arrows next to the post.  In general, vote up content that is clear, interesting, well-supported, and, for answers, correct.  Vote down content that is unclear or demonstrably wrong.
+
+**Voting is not a popularity contest.**  Don't vote based on who posted something -- don't vote for your friends or vote down posts from people you don't like.  
+
+**Voting is not to express personal opinions or preferences.**  Vote objectively, not subjectively.  On a programming community, don't downvote an answer because it recommends using a library or tool you hate; vote based on whether that library or tool fits the question.  On a religion community, don't upvote a post just because it's from the "right" denomination.  **Evaluate posts based on their content**, and evaluate answers in the context of the questions they answer.
+
+
+# Are there limits on voting?
+
+If you have not yet earned the Participate Everywhere ability, you can vote only on answers to your own questions.  After you earn this ability, you can vote on any post.
+
+There is a daily limit on the number of votes you can cast.  This limit can be changed per community; the default is 30.
+
+
+# When should I change my vote?
+
+You can remove or reverse your vote at any time.  If you downvoted and the author edits to address your objection, you should at least remove your downvote (and upvote if you now find it appropriate).  If you've reconsidered -- maybe you have more information now than when you first voted -- then you can change your vote.  You aren't locked in; if you no longer agree with one of your past votes, change it.
+
+
+# What else can I do?
```

Comment:
imo this is one of the most important and most neglected pieces for downvoting context. Can we move this section higher or highlight it somehow?

Reply:
Moved up.